tests/src/Functional/ErrorHandlerTest.php

```diff
@@ -78,8 +83,6 @@ public function testErrorPages() {
     ];
 
     $routes = [
-      'apigee_edge_test.entity_storage_exception',
```

Comment:
@arlina-espinoza Should we still test these routes? Or are we completely replacing `apigee_edge_test`?